lib/faker/default/verb.rb

```diff
@@ -3,22 +3,67 @@
 module Faker
   class Verb < Base
     class << self
+      ##
+      # Produces the base form of a verb.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Verbs.base #=> "shape"
```

Comment:
Need to change these to Faker::Verb
```suggestion
      #   Faker::Verb.base #=> "shape"
```